packages/react-ui/src/GlobalStyles/components/Links.js

```diff
@@ -44,14 +44,14 @@ export default createGlobalStyle`
     position: relative;
     z-index: -1;
     display: block;
-    height: 11rem; /* Fixed header's height */
-    margin-top: -11rem; /* Fixed header's negative height */
+    height: 14rem; /* Fixed header's height */
```

Comment:
the header is larger with the headband (forogt to change it before)

Reply:
I totally miss what that thing should do. could you add a comment to explain the purpose of `:target::before` ?

Reply:
::target c'est pour cibler l'élément qui a l'id qui correspond au # dans l'url, très pratique ! Ensuite je rajoute une marge qui correspond au header fixed pour par que le titre se retrouve caché dessous